### PR TITLE
Update content/es/docs/tasks/tools

### PR DESCRIPTION
**Pull request by leeyspaul**
Spanish translation of content/es/docs/tasks/tools

Parent: #13948

### CODE REVIEW

**Comment by leeyspaul** [comment]
ptal @raelga I closed the last PR as it got really messy and started a new here. I'm seeing a block due to the newest release but please let me know if there's anything I should fix in the meantime!

**Review by sftim — CHANGES_REQUESTED** [review]
How come `content/es/docs/tasks/tools/_index.md` is in English? That doesn't look right.

**Comment by leeyspaul** [comment]
Yeah, sorry about that. Difficulty focusing yesterday haha, thanks for catching it! Just pushed up the Spanish files now.

File: content/es/docs/tasks/tools/install-kubectl.md [hunk 1]
```diff
@@ -49,65 +82,146 @@ yum install -y kubectl
...
+
+    ```bash
+    kubectl version --client
     ```
 
-## Instalar con Powershell desde PSGallery
+{{< note >}}
+[Docker Desktop para Windows](https://docs.docker.com/docker-for-windows/#kubernetes) añade su propia versión de `kubectl` a PATH.
+
+Si tienes Docker Desktop instalado, es posible que tengas que modificar tu PATH al PATH añadido por Docker Desktop o remover `kubectl` de Docker Desktop.
```
Comment (raelga) [inline content/es/docs/tasks/tools/install-kubectl.md:213]:
```suggestion
Si tienes Docker Desktop instalado, es posible que tengas que modificar tu PATH al PATH añadido por Docker Desktop o eliminar la versión de `kubectl` proporcionada por Docker Desktop.
```

File: content/es/docs/tasks/tools/install-kubectl.md [hunk 2]
```diff
@@ -22,21 +20,56 @@ Debes usar una versión de kubectl que esté a menos de una versión menor de di
...
+4. Comprueba que la versión que se ha instalado es la más reciente.
+
+    ```
+    kubectl version --client
+    ```
 
 Estos son algunos métodos para instalar kubectl.
 
-## Instalar el binario de kubectl usando gestión nativa de paquetes
+## Instalar usando gestión nativa de paquetes
```
Comment (raelga) [inline content/es/docs/tasks/tools/install-kubectl.md:61]:
```suggestion
## Instalar mediante el gestor de paquetes del sistema
```

File: content/es/docs/tasks/tools/install-kubectl.md [hunk 3]
```diff
@@ -49,65 +82,146 @@ yum install -y kubectl
...
-    ```
-    kubectl version
+    ```bash
+    kubectl version --client
     ```
 
-## Instalar con Macports en macOS
+### Instalar con Macports en macOS
 
 Si estás en macOS y usando el gestor de paquetes [Macports](https://macports.org/), puedes instalar kubectl con Macports.
```
Comment (raelga) [inline content/es/docs/tasks/tools/install-kubectl.md:173]:
```suggestion
Si estás en macOS y utilizas el gestor de paquetes [Macports](https://macports.org/), puedes instalar `kubectl` con `port`.
```

File: content/es/docs/tasks/tools/install-kubectl.md [hunk 4]
```diff
@@ -49,65 +82,146 @@ yum install -y kubectl
...
 
 1. Ejecuta los comandos de instalación:
 
-    ```
+    ```bash
     sudo port selfupdate
     sudo port install kubectl
     ```
 
 2. Para asegurar que la versión utilizada sea la más actual puedes probar:
```
Comment (raelga) [inline content/es/docs/tasks/tools/install-kubectl.md:182]:
```suggestion
2. Para asegurar que la versión utilizada sea la más actual puedes ejecutar:
```

File: content/es/docs/tasks/tools/install-kubectl.md [hunk 5]
```diff
@@ -49,65 +82,146 @@ yum install -y kubectl
...
-## Instalar con Powershell desde PSGallery
+{{< note >}}
+[Docker Desktop para Windows](https://docs.docker.com/docker-for-windows/#kubernetes) añade su propia versión de `kubectl` a PATH.
+
+Si tienes Docker Desktop instalado, es posible que tengas que modificar tu PATH al PATH añadido por Docker Desktop o remover `kubectl` de Docker Desktop.
+{{< /note >}}
+
+### Instalar con Powershell desde PSGallery
 
 Si estás en Windows y usando el gestor de paquetes [Powershell Gallery](https://www.powershellgallery.com/), puedes instalar y actualizar kubectl con Powershell.
```
Comment (raelga) [inline content/es/docs/tasks/tools/install-kubectl.md:218]:
```suggestion
Si estás en Windows y utilizas el gestor de paquetes [Powershell Gallery](https://www.powershellgallery.com/), puedes instalar y actualizar kubectl con Powershell.
```

File: content/es/docs/tasks/tools/install-kubectl.md [hunk 6]
```diff
@@ -175,107 +294,32 @@ Puedes instalar kubectl como parte del Google Cloud SDK.
...
-
+    ```shell
+    kubectl version --client
+    ``
 
-## Configurar kubectl
+## Comprobar la configuración kubectl
 
-Para que kubectl pueda encontrar y acceder a un clúster de Kubernetes, necesita un [fichero kubeconfig](/docs/tasks/access-application-cluster/configure-access-multiple-clusters/), que se crea de forma automática cuando creas un clúster usando [kube-up.sh](https://github.com/kubernetes/kubernetes/blob/master/cluster/kube-up.sh)  o despliegas de forma satisfactoria un clúster de Minikube. Revisa las [guías para comenzar](/docs/setup/) para más información acerca de crear clústers. Si necesitas acceso a un clúster que no has creado, ver el [documento de Compartir Acceso a un Clúster](/docs/tasks/access-application-cluster/configure-access-multiple-clusters/).
+Para que kubectl pueda encontrar y acceder a un clúster de Kubernetes, necesita un [fichero kubeconfig](/docs/tasks/access-application-cluster/configure-access-multiple-clusters/), que se crea de forma automática cuando creas un clúster usando [kube-up.sh](https://github.com/kubernetes/kubernetes/blob/master/cluster/kube-up.sh)  o despliegas de forma satisfactoria un clúster de Minikube.
```
Comment (raelga) [inline content/es/docs/tasks/tools/install-kubectl.md:309]:
```suggestion
Para que kubectl pueda encontrar y acceder a un clúster de Kubernetes, necesita un [fichero kubeconfig](/docs/tasks/access-application-cluster/configure-access-multiple-clusters/), que se crea de forma automática cuando creas un clúster usando [kube-up.sh](https://github.com/kubernetes/kubernetes/blob/master/cluster/kube-up.sh)  o despliegas de forma satisfactoria un clúster de Minikube. Revisa las [guías para comenzar](/docs/setup/) para más información acerca de crear clústers. Si necesitas acceso a un clúster que no has creado, ver el [documento de Compartir Acceso a un Clúster](/docs/tasks/access-application-cluster/configure-access-multiple-clusters/).
```

File: content/es/docs/tasks/tools/install-kubectl.md [hunk 7]
```diff
@@ -323,16 +369,23 @@ Debes asegurarte que la secuencia de comandos de completado de kubectl corre en
...
     ```
 
 - Añade la secuencia de comandos de completado al directorio `/etc/bash_completion.d`:
 
-    ```shell
+    ```bash
     kubectl completion bash >/etc/bash_completion.d/kubectl
     ```
 
+Si tienes alias para kubectl, puedes extender los comandos de shell para funcionar con ese alias:
```
Comment (raelga) [inline content/es/docs/tasks/tools/install-kubectl.md:382]:
```suggestion
Si tienes un alias para `kubectl`, puedes extender los comandos de shell para funcionar con ese alias:
```

**Review by raelga — CHANGES_REQUESTED** [review]
@leeyspaul thanks for updating the tools section!

The review addresses some weird wording (some parts looked like generated by an automated translation) and some typos.

Feel free to ping me on slack if you have any question regarding the review!

**Comment by leeyspaul** [comment]
Thank you for those. Spanish is a bit rusty haha. I appreciate the fixes and feedback! Please let me know if there is anything else. @raelga

**Comment by raelga** [comment]
Thanks @leeyspaul! :D

Can you rebase your branch into a single commit?

**Comment by leeyspaul** [comment]
Ok, I rebased into a single commit. Sorry it's a bit messy haha. Please let me know if there's anything else I need to fix!

File: content/es/docs/tasks/tools/install-kubectl.md [hunk 8]
```diff
@@ -175,107 +294,32 @@ Puedes instalar kubectl como parte del Google Cloud SDK.
...
-    Para averiguar la última versión estable (por ejemplo, para secuencias de comandos), echa un vistazo a [https://storage.googleapis.com/kubernetes-release/release/stable.txt](https://storage.googleapis.com/kubernetes-release/release/stable.txt).
-
-2. Añade el binario a tu PATH.
-{{% /tab %}}
-{{< /tabs >}}
-
-
+    ```shell
+    kubectl version --client
+    ``
```
Comment (raelga) [inline content/es/docs/tasks/tools/install-kubectl.md:305]:
```suggestion
    ```
```

Just a missing back-tick.

File: content/es/docs/tasks/tools/install-kubectl.md [hunk 9]
```diff
@@ -116,57 +230,62 @@ Si estás en Windows y usando el gestor de paquetes [Powershell Gallery](https:/
...
+### Instalar en Windows usando Chocolatey o scoop
 
-Para instalar kubectl en Windows puedes usar bien el gestor de paquetes [Chocolatey](https://chocolatey.org) o el instalador de línea de comandos [scoop](https://scoop.sh).
-{{< tabs name="kubectl_win_install" >}}
-{{% tab name="choco" %}}
+1. Para instalar kubectl en Windows puedes usar el gestor de paquetes [Chocolatey](https://chocolatey.org) o el instalador de línea de comandos [scoop](https://scoop.sh).
 
+    {{< tabs name="kubectl_win_install" >}}
+    {{% tab name="choco" %}}
+    ```powershell
```
Comment (raelga) [inline content/es/docs/tasks/tools/install-kubectl.md:246]:
```suggestion
Using  [Chocolatey](https://chocolatey.org).

    ```powershell
```

Let's try if adding a text line fixes the rendering issue:

Screenshot 2020-12-16 at 15 08 09

https://deploy-preview-25481--kubernetes-io-master-staging.netlify.app/es/docs/tasks/tools/install-kubectl/#instalar-en-windows-usando-chocolatey-o-scoop

File: content/es/docs/tasks/tools/install-kubectl.md [hunk 10]
```diff
@@ -116,57 +230,62 @@ Si estás en Windows y usando el gestor de paquetes [Powershell Gallery](https:/
...
     choco install kubernetes-cli
-
-{{% /tab %}}
-{{% tab name="scoop" %}}
-
+    ```
+    {{% /tab %}}
+    {{% tab name="scoop" %}}
+    ```powershell
```
Comment (raelga) [inline content/es/docs/tasks/tools/install-kubectl.md:251]:
```suggestion
Using [scoop](https://scoop.sh).

    ```powershell
```

Let's try if adding a text line fixes the rendering issue:
image
https://deploy-preview-25481--kubernetes-io-master-staging.netlify.app/es/docs/tasks/tools/install-kubectl/#instalar-en-windows-usando-chocolatey-o-scoop

**Review by raelga — CHANGES_REQUESTED** [review]
Thanks @leeyspaul, last change request, I promise!

File: content/es/docs/tasks/tools/install-kubectl.md (hunk 9 above)
Comment (leeyspaul) [reply content/es/docs/tasks/tools/install-kubectl.md:246]: Yeah, weird! Thank you for catching this.

**Comment by leeyspaul** [comment]
Do let me know if there is anything else :]

**Comment by raelga** [comment]
Thanks @leeyspaul, the tab "glitch" is still there, but the raw code looks good (the same as the properly rendered english version). Let's merge it and see if it works, is only aesthetics and the content looks good!

/approve
/lgtm

**Pull request merged**